site/specs/numbers.json

```diff
@@ -11307,110 +11307,109 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieves the removeImportedTnOrders requests for the given account ID. <p>A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.</p></p>\n",
-        "operationId": "GET /accounts/{accountId}/removeImportedTnOrders",
+        "description": "Retrieves the Remove Imported Tn Orders requests for the given account ID.\nA maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.",
```

Comment:
The new docsite doesn't use a `*` to indicate required fields.  It now looks like this 
<img width="661" alt="image" src="https://user-images.githubusercontent.com/2125458/164010289-d57ca076-00f4-4ac3-9433-a3522bd4a726.png">